exercises/largest-series-product/example.cr

```diff
@@ -1,14 +1,13 @@
 class Series
+  @series : Array(Int64)
```

Comment:
It's my understand that this is necesary now that the line reading `@series = [] of Int64` is removed. Removing it would cause something like https://travis-ci.org/exercism/xcrystal/builds/150355827.